draft-packages/modal/KaizenDraft/Modal/Presets/ConfirmationModal.elm

```diff
@@ -207,24 +207,29 @@ header config =
     let
         resolveIcon =
             case config.variant of
-                Positive ->
-                    svgAsset "@kaizen/component-library/icons/success.icon.svg"
+                Cautionary ->
+                    svgAsset "@kaizen/draft-modal/KaizenDraft/Modal/illustrations/cautionary.icon.svg"
```

Comment:
I'm kinda surprised there's no `hosted-assets` function for Elm here... I wonder if there's a reason for this? It would be a little nicer than duplicating assets. cc/ My go-to Elm person @mbylstra 

https://github.com/cultureamp/kaizen-design-system/blob/master/packages/hosted-assets/index.ts#L12

Reply:
Ah, I did actually make a hosted-asset Elm function in our repo last cycle which I intended to move into the Kaizen in cooldown, but it didn't happen. I'm planning on dedicating some weekly time to Kaizen - this would be a good task. So, I can refactor this later in a separate PR once a hosted asset function exists.

Reply:
I knew you would know :D That would be a huge win.

Reply:
Just good timing really (or bad timing because i didn't get it in in time)